server/bin/test/conftest.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+import shutil
+import tempfile
+from pathlib import Path
+
+from pbench.server.api import get_server_config
+
+server_cfg_tmpl = """[DEFAULT]
```

Comment:
We haven't really moved toward click-ifying the server jobs; and there's less value since they're not intended for interactive use. But that arguably limits the ability to leverage the testing tree under `lib/pbench`.

In general, I'd argue for putting as much of the testable code in the library modules and leave very little in `server/bin`, but right now that may not be entirely practical. I'm not sure if there's any way to make pytest look for a `conftest.py` file outside the immediate test treee? If so, that's a possibility, because I agree the duplication is distasteful and potentially error prone.

Reply:
I looked into this that if we can place the conftest.py file outside the immediate test trees but it doesn't seem to be possible at present.